spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'create user', type: :system do
```

Comment:
Обычно `describe` используется для описания классов либо его методов. Подробнее можно прочитать про это здесь: https://www.betterspecs.org/#describe

Чтобы описать какую-то фичу, то принято использовать `Rspec.feature`
```suggestion
RSpec.feature 'CreateUser' do
```